phpBB/posting.php

```diff
@@ -1525,9 +1525,13 @@
 			'L_MAX_VOTES'		=> $user->lang('MAX_OPTIONS_SELECT', (int) $post_data['poll_max_options']),
 		));
 
-		$parse_poll->message = implode("\n", $post_data['poll_options']);
-		$parse_poll->format_display($post_data['enable_bbcode'], $post_data['enable_urls'], $post_data['enable_smilies']);
-		$preview_poll_options = explode('<br />', $parse_poll->message);
+		$preview_poll_options = array();
```

Comment:
Each poll option has to be parsed/rendered individually. The old code relies on the assumption that newlines are replaced with `<br />` exactly, which may not always be true depending on the text formatter. And in this case, s9e\TextFormatter replaces newlines with `<br>`.

IIRC there are also some edge cases where you can start a BBCode in one poll option and end it in another, and end up with a broken layout.